stable/voyager/Chart.yaml

```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 description: 'Voyager by AppsCode - Secure Ingress Controller for Kubernetes'
+home: https://cdn.appscode.com/
```

Comment:
Please set this to `https://appscode.com/products/voyager/`

Reply:
OK, it's done.